app/controllers/users_controller.rb

```diff
@@ -8,6 +8,15 @@ class UsersController < ActionController::Base
         :email_exists => "email already exists"
     }
 
+    def matches
+        user = User.find_by_id(params[:id])
+        if user.present?
+            render :json => user.matches
+        else
+            render :json => []
```

Comment:
Also, AFAIK all of our API endpoints use `/:id` (e.g. `/api/v1/profile/:id`) instead of putting the id in `params` (e.g. `/api/v1/profile?id=:id`).  I'd suggest changing this to keep things consistent.